clang/include/clang/Basic/DiagnosticGroups.td

```diff
@@ -1316,6 +1316,9 @@ def CudaUnknownVersion: DiagGroup<"unknown-cuda-version">;
 // ignored by CUDA.
 def HIPOnly : DiagGroup<"hip-only">;
 
+// A warning used when compiling for HIP or CUDA without linking LIBSPIRV.
```

Comment:
A warning generated when compiling SYCL applications for HIP or CUDA backends without linking LIBSPIRV